tests/integ/test_stored_procedure.py

```diff
@@ -1171,7 +1177,7 @@ def test_anonymous_stored_procedure(session):
     assert add_sp(1, 2) == 3
 
 
-@pytest.mark.skipif(IS_IN_STORED_PROC, reason="need resources")
+@pytest.mark.skipif(IS_NOT_ON_GITHUB, reason="need resources")
```

Comment:
Don't you still need `IS_IN_STORED_PROC`?

Reply:
Ohhh is it because `IS_NOT_ON_GITHUB` assumes `IS_IN_STORED_PROC`?

Reply:
you're right, they are mutually exclusive